src/features/embalm/stepContent/steps/SelectArchaeologists.tsx

```diff
@@ -73,7 +60,7 @@ export function SelectArchaeologists() {
       direction="column"
       width="100%"
     >
-      <Heading>Archaeologists</Heading>
+      {!hideHeader ?? <Heading>Archaeologists</Heading>}
```

Comment:
I realised when resolving merge conflicts that the thing this was solving no longer exists. This PR will bring it back (ie the extra Heading on the Archaeologists page), but we can revert this later.